src/components/DotIndicatorMessage.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import styles from '../styles/styles';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import colors from '../styles/colors';
+import variables from '../styles/variables';
+import Text from './Text';
+
+const propTypes = {
+    /** The messages to display  */
+    // eslint-disable-next-line react/forbid-prop-types
+    messages: PropTypes.object,
```

Comment:
Why is this a forbidden prop type and need the eslint-disable?

Reply:
Oh I see it's b/c of how this was built previously in https://github.com/Expensify/App/blob/3e33ea35536c831de12474d7a027bd55c45ce353/src/components/OfflineWithFeedback.js#L27-L29

If I'm not mistaken `sortedErrors` is an array so let's fix this here and now and leave the code hopefully in a little bit better place than we found it.

Making this an array will mean we should always pass an array then to this component which you're already doing anyways here: https://github.com/Expensify/App/blob/3e33ea35536c831de12474d7a027bd55c45ce353/src/pages/signin/ResendValidationForm.js#L74-L79